src/main/java/hudson/plugins/jira/Updater.java

```diff
@@ -68,65 +68,65 @@ public Updater(SCM scm, List<String> labels) {
         }
     }
 
-    boolean perform(Run<?, ?> build, TaskListener listener, AbstractIssueSelector selector) {
+    boolean perform(Run<?, ?> run, TaskListener listener, AbstractIssueSelector selector) {
```

Comment:
why renaming this? it's confusing as it generate some noise for nothing. (i.e make me feeling what happen here a lot of changes!)